sdk/communication/azure-communication-chat/README.md

```diff
@@ -239,9 +239,9 @@ listMessages returns different types of messages which can be identified by `cha
 
 -`ThreadActivity/TopicUpdate`: System message that indicates the topic has been updated.
 
--`ThreadActivity/AddMember`: System message that indicates one or more members have been added to the chat thread.
```

Comment:
@danielgerlag These system messages need to be left as they are for now because they are used in other systems i.e trouter / cns  (we are not the producers of them). There is another feature that will tackle this update across all of them.